docs/getting-started/install-configure.md

```diff
@@ -137,15 +137,15 @@ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.
 {{% tab "Latest" %}}
 
 ```bash
-curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | CHANNEL=master sh
+curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | CXP_CHANNEL=master sh
```

Comment:
I think this is a great idea 👍🏻 what would you think of just using `XP` instead of `CXP` given that `XP` is the abbreviation we use in other contexts?

Reply:
I'm good with either. Went with `CXP` initially as that's what the discussion at the issue report suggested but happy to change it to just `XP` for cross-plane.